lib/translations_checker/locale_file_content.rb

```diff
@@ -0,0 +1,30 @@
+require "translations_checker/locale_file"
+
+require "yaml"
+require "fossicker"
+
+module TranslationsChecker
+  class LocaleFileContent
+    using Fossicker
+
+    attr_reader :yaml
+
+    def initialize(yaml)
+      @yaml = yaml
+    end
+
+    def [](key_path)
+      content.fossick(*key_path, default: nil)
```

Comment:
@johncarney I like your `fossicker` gem as a generalization of `.dig` for other collections, but since we're only dealing with Hashes here can we just use `.dig`? Standard library methods are easier to read as there is a great chance that the reader already knows what the method is doing :)

Reply:
I used `dig` originally, but ...
```ruby
> { key: "value" }.fossick(*[])
# => { key: "value" }
> { key: "value" }.dig(*[])
# => ArgumentError: wrong number of arguments (given 0, expected 1+)
```